src/isPalindrome.js

```diff
@@ -20,6 +20,22 @@
  */
 function isPalindrome(str) {
   // write code here
+  const regStr = /\w/g;
+
+  if (str === '') {
+    return true;
+  }
+
+  const lowerStr = (str.toLowerCase()).match(regStr);
```

Comment:
методы выполняются в том порядке в котором написаны, скобки не нужны
```suggestion
  const letters = str.toLowerCase().match(regStr);
```